lib/helper/WebDriver.js

```diff
@@ -2055,6 +2055,34 @@ class WebDriver extends Helper {
     return this.browser.getGeoLocation();
   }
 
+  /**
+   * {{> grabElementBoundingRect }}
+   *
+   */
+  async grabElementBoundingRect(locator, prop) {
+    const res = await this._locate(locator, true);
+    assertElementExists(res, locator);
+
+    const arr1 = await forEachAsync(res, async el => el.getLocation());
+    const arr2 = await forEachAsync(res, async el => el.getSize());
+    const arr = [];
+    for (let index = 0; index < arr1.length; index++) {
+      arr.push(Object.assign({}, arr1[index], arr2[index]));
+    }
+    const result = [];
+    if (prop) {
+      arr.forEach((element) => {
+        result.push(element[prop]);
+      });
+    } else {
+      arr.forEach((element) => {
+        result.push(element);
+      });
+    }
+
+    return result;
```

Comment:
You can use `find` for find element with prop and immediately return the result, like
```
if (prop) {
      return [arr.find((element) => {...});]
    }
```

If prop is null - return els.